k8s/bases/swe-d2cc/swe-d2cc-cgm-minio-automatic-purge.yaml

```diff
@@ -0,0 +1,36 @@
+kind: CronJob
+apiVersion: batch/v1
+metadata:
+  name: swe-d2cc-cgm-minio-automatic-purge
+  labels:
+    app: gridcapa
+    component: cgm-minio-automatic-purge
+    region: swe
+    process: d2cc
+spec:
+  schedule: "0 0 * * *"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 3
+  successfulJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: swe-d2cc-cgm-minio-automatic-purge
+              image: minio-mc
+              args: [ "rm", "-r", "--force", "--older-than", "7d", "minio/gridcapa/SWE/D2CC/CGMs/" ]
```

Comment:
Can we make several orders in the same cronjob? I think is better to have one cronjob for swe-d2cc that delete different kind of files  . What do you think ?

Reply:
I think we can have several containers in the same file, one for each cronjob, if it's ok for you, a can make a file for cronjobs idcc, and a file for d2cc